Polyhedron/demo/Polyhedron/Plugins/AABB_tree/Cut_plugin.cpp

```diff
@@ -1204,14 +1204,33 @@ void Polyhedron_demo_cut_plugin::createCutPlane() {
     CGAL::Three::Scene_item* item = scene->item(i);
     Scene_polyhedron_item* poly_item = qobject_cast<Scene_polyhedron_item*>(item);
     if(!poly_item) continue;
-
+    if(!poly_item->polyhedron()->is_pure_triangle())
+    {
+      messages->warning(QString("%1 ignored (not pure triangle)").arg(poly_item->name()));
+      continue;
+    }
     if(facet_trees.find(poly_item) == facet_trees.end()) {
       facet_trees[poly_item] = new Facet_tree();
       PPMAP pmap;
-      facet_trees[poly_item]->insert(faces(*(poly_item->polyhedron())).first,
+      //filter facets to ignore degenerated ones
+      for(Polyhedron::Facet_iterator fit = poly_item->polyhedron()->facets_begin(), end = poly_item->polyhedron()->facets_end(); fit!=end; ++fit)
```

Comment:
This line of code is far too long. According to our coding guidelines, the line width should be limited to 80 columns. There is a tolerance, but you should try to reduce the width of your lines. For this, `for` loop, it can be reduced easily:
```C++
for(Polyhedron::Facet_iterator fit = poly_item->polyhedron()->facets_begin(),
                               end = poly_item->polyhedron()->facets_end();
      fit!=end; ++fit)
```